pyproject.toml

```diff
@@ -11,6 +11,7 @@ python = ">=3.8,<3.12"
 opshin = "^0.12.5"
 pycardano = "^0.8.0"
 click = "^8.1.3"
+pyaiken = "^0.4.0"
```

Comment:
I would consider making this an optional dependency, because it basically adds the whole Rust toolchain as a dependency (which has caused errors earlier)